src/main/scala/com/fulcrumgenomics/commons/collection/BetterBufferedIterator.scala

```diff
@@ -26,6 +26,8 @@ package com.fulcrumgenomics.commons.collection
 
 import com.fulcrumgenomics.commons.CommonsDef._
 
+import scala.collection.BufferedIterator
```

Comment:
?

Reply:
I think scala.BufferedIterator got removed as part of the collections re-org, so now you have to import it specifically.